packages/client/src/app/jobs/executeJobModal.tsx

```diff
@@ -20,19 +27,17 @@ export const ExecuteJobModal = ({ closeModal, devices, jobId }: ExecuteJobModalP
           if (response.status !== 200) {
             throw new Error();
           }
-
           closeModal();
         },
       );
 
       toast.promise(promise, {
         pending: `Running ${jobId}...`,
-        success: `${jobId} started succesfully`,
+        success: `${jobId} started successfully`,
         error: `Failed to start job ${jobId}`,
       });
     },
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    [jobId],
+    [jobId, closeModal], // Abhängigkeiten
```

Comment:
What does this comment mean?

Reply:
Dependencies, German word

Reply:
lol, in a future... use English in all the code you write, unless it's a "dupa" then it's fine.